meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -498,6 +498,16 @@ UBOOT_SUFFIX:imx93-11x11-lpddr4x-evk = "bin"
 UBOOT_DTB_NAME:imx93-11x11-lpddr4x-evk ?= "imx93-11x11-evk.dtb"
 
 # STM32MP1
+### STM32 custom image.bin symlink
+stm32_imagebin_symlink() {
+       ln -sf ${LMP_FLASHLAYOUT_IMAGE} ${IMGDEPLOYDIR}/${MFGTOOL_FLASH_IMAGE}.bin
+}
+
+STM32_IMAGEBIN_SYMLINK = ""
+STM32_IMAGEBIN_SYMLINK:stm32mp1common = "stm32-imagebin-symlink"
```

Comment:
We might want to have this named in a bit more generic way in the future but this is fine for now.

Reply:
symlink is needed only for flashlayout support